### ISSUE TITLE
Unable to fetch "IEC/PAS 62596"

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-iec#31]
https://webstore.iec.ch/publication/20742

But:
<img width="823" alt="Screen Shot 2020-12-21 at 5 07 32 PM" src="https://user-images.githubusercontent.com/11865/102759397-0492cf00-43af-11eb-8aa2-9904650e1f3f.png">

So we should accept the syntax of "IEC/PAS XXXX" as well.

**Comment by ronaldtse** [comment]
Check if this has changed with #47 .

**Comment by andrew2net** [comment]
@ronaldtse the Harmonized API provides the document with `IEC PAS 62596` ID.  So to fetch it we should use:
```sh
$ relaton fetch --no-cache "IEC PAS 62596"
[relaton-iec] (IEC PAS 62596) Fetching from Relaton repsitory ...
[relaton-iec] (IEC PAS 62596) Found: `IEC PAS 62596:2009`
<bibdata type="standard" schema-version="v1.2.5">
...
```
I seems slash used to divide copublishers `IEC/ISO`, `IEC/IEEE` etc.

**Comment by ronaldtse** [comment]
I see. So I actually tried that after the first tried failed. However it says that the cache says it’s not found? Let me try again.

**Comment by andrew2net** [comment]
Yes, because in the relaton cache filenames are created from references. Slashes and spaces replaced with underscores. So the requests `IEC PAS 62596` and `IEC/PAS 62596` are stored in same file. It needs to clear DB by `relaton DB clear`.

Maybe we should replace slashes with hyphens?
